it/okhttp/build.gradle

```diff
@@ -1,4 +1,4 @@
 dependencies {
-    testImplementation 'com.squareup.okhttp3:okhttp'
-    testImplementation 'com.squareup.okhttp3:okhttp-tls'
+    testImplementation 'com.squareup.okhttp3:okhttp:4.10.0'
```

Comment:
How about 4.10.0 in dependencies.yml and 3.14.9 in retrofit project? 3.x is EOL for a long time so there isn't any chance of version drift for it, so it doesn't have to be in an easy-to-update location

Reply:
Thanks, good point 👍  Done.